app/controllers/mobile.js

```diff
@@ -41,21 +45,31 @@ export default class MobileController extends Controller {
       }
     } else {
       // will cancel the login
+      try {
+        const checkStatus = await this.fetchAuthStatus(AUTH_STATUS.REJECTED);
+        if (checkStatus.response.status !== 200) {
+          throw Error('Something went wrong. Please try again later');
+        }
+        this.toast.success('Your request has been cancelled', 'Success');
+      } catch (error) {
+        this.toast.error(
+          'Something went wrong. Please try again later',
+          '',
+          toastNotificationTimeoutOptions
+        );
+      }
     }
   }
 
   @action async buttonClicked() {
     try {
-      const response = await fetch(
-        `${ENV.BASE_API_URL}/auth/qr-code-auth-get-device-info?user_id=${this.model.userId}`,
-        {
-          method: 'GET',
-          headers: {
-            'Content-Type': 'application/json',
-          },
-          credentials: 'include',
-        }
-      );
+      const response = await fetch(`${ENV.BASE_API_URL}/auth/device`, {
```

Comment:
i think we can also create a constant for this URL so we can change all the URLs from a single place, managing the code as it grows will become easier.